scipy-1.0/paper.tex

```diff
@@ -28,7 +28,19 @@
 \keywords{Scientific computing, Python, Mathematics}
 
 \begin{abstract}
-TODO. Abstract must be under 200 words and not include subheadings or citations.
+SciPy is an open source scientific computing library for the Python programming language.
+SciPy 1.0 was released in late 2017, about 16 years after the original
+version 0.1 release. SciPy
+has become a \emph{de facto} standard in the community, with more than 600
```

Comment:
How about adding number of dependent packages and repositories, as calculated by https://libraries.io/pypi/scipy?